bats/tests/containers/factory-reset.bats

```diff
@@ -156,28 +156,28 @@ rdctl_factory_reset() {
 
 check_directories() {
     # Check if all expected directories are created after starting application/ are deleted after a factory reset
-    delete_dir=("$PATH_APP_HOME" "$PATH_CONFIG")
+    delete_dir=("$PATH_CONFIG")
```

Comment:
From the comments on this issue, I understand that for the bats tests, we are assuming that the snapshots directory is not present until the snapshots feature is more complete. So shouldn't we just use the old test code?

Reply:
For the record, I went back to `main` and `RD_LOCATION=dev bats bats/tests/containers/factory-reset.bats` fails on macOS on `main`, passes on this branch.